source/developers-guide/shopware-5-upgrade-guide-for-developers/index.md

```diff
@@ -17,14 +17,205 @@ including minor and bugfix releases, refer to the `UPGRADE.md` file found in you
+
+The following were made to implement this feature:
+
+* Added `Shopware\Components\Cart\ProportionalTaxCalculator` to calculate proportional taxes for the cart items
+* Added `Shopware\Components\Cart\BasketHelper` to to add items to the cart that need to be calculation in a proportional way
+* Added `Shopware\Components\Cart\ProportionalCartMerger` to merge proportional cart items into one cart item
+* Added new filter event to modify proportional vouchers `Shopware_Modules_Basket_AddVoucher_VoucherPrices`   
+* Added new column `invoice_shipping_tax_rate` to s_order, to save exact dispatch shipping tax rate.
+
+### elasticsearch
+
+Shopware 5.5 supports elasticsearch versions 2.x, 5.x and 6.x.
+
+Shopware works with index aliases to allow multiple indices to exist in parallel and switch between them if necessary.
+Should an index with the name of an alias (e.g. `shop_1`) already exist, will now be deleted so that the new alias can
+be created without error.
+
+To support elasticsearch 6 it was necessary to split the existing index into multiple indices for different document types
+(product, property). If you're using `sw:es:analyze` and `sw:es:switch:alias`, you now need to also provide the parameter
+document type that you want to analyze or switch the index of.
+
+### elasticsearch backend
+
+A new `EsBackend` bundle was added to index and search for products, customers and orders in the backend. New config.php
+parameters where added for that, the `es` array of parameters now contains a new key `backend`:
+
+```
+'es' => [
+    ...
+    'backend' => [
+        'write_backlog' => false,
+        'enabled' => false,
+    ],
+    ...
+],
+```
+
+To activate elasticsearch in the backend, simply change `backend.enabled` to `true` and use the `sw:es:backend:index:populate`
+command to populate the necessary indices. If you also enabled the `backend.write_log`, you can use `sw:es:backend:sync`
+periodically after that to keep the index in sync.
+
+### Changed execution model of `replace` hooks
+
+When multiple `replace` hooks exist for one method, up to 5.4 (incl.) each of these hooks were called sequentially with
+each of these hooks having the opportunity to call `executeParent()`. Hence, if this happened, the original (parent) 
+implementation got called multiple times, leading to unexpected behaviours. 
+
+In Shopware 5.5 the execution model was changed to a decorative type of implementation: If more than one `replace` hook
+exists for a function, calling `executeParent()` inside the hook will execute the next `replace` hook of function. Only
+the last `replace` hook will then call the original implementation on `executeParent()`.
+ 
+### Filesystem abstraction layer
+
+As of Shopware v5.5, the use of direct file system access methods (like e.g. `fopen()`, `file()`, `is_readable()` and many more)
```

Comment:
we don't add the version prefix, please use `5.5`

Reply:
okilidokili!